installing.adoc

```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
 4. `Use {product-short}'s package repository [Y]` +
 displays, press `enter` to continue. Your system will be configured to add
 the {product-short} packaging repository for *yum* or *apt-get* as appropriate
-so it can install the {product-short} 3rd party packages.
+so it can install the {product-short} 3rd party packages. 
+*{product-short} IMAPD is meant to be installed on a multi-server environment, with
```

Comment:
@jorgedlcruz Here and anywhere else where you have bulleted or numbered list items, please be sure to have a single space between the marker and the text that follows.

Reply:
so it can install the {product-short} 3rd party packages.